DeUrgenta/Features/CreateGroup/View/AddGroupNameView.swift

```diff
@@ -48,6 +54,9 @@ struct AddGroupNameView: View {
                     .cornerRadius(6)
                 })
             }
+            .alert(isPresented: $showingAlert, content: {
+                Alert(title: Text("Important"), message: Text("Denumirea grupului este obligatorie."), dismissButton: .default(Text("OK")))
```

Comment:
Can we localize the alert text, as we have for other strings?

Reply:
Yes, of course :)